src/ybadge.h

```diff
@@ -24,22 +19,76 @@
 #define BUTTON3_PIN 13
 
 ////////////////////////////// LEDs ///////////////////////////////////////////
+
+/*
+*  This function is only used behind the scenes to configure the LEDs and should not be used in your code.
+*/
 void leds_init();
+
+/*
+*  This function sets the brightness of all of the LEDs. 
+*  The brightness is specified as an integer between 0 (off) and 255 (full brightness).
+*  The void return type means that this function does not return a value.
+*/
 void leds_set_brightness(uint8_t brightness);
+
+/*
+*  This function sets the color of a single LED.
+*  The index is an integer between 1 and 20, representing the number of the target LED (for example, 1 corresponds to LED 1 on the board).
+*  The red, green, and blue values are integers between 0 (off) and 255 (full brightness).
+*  The void return type means that this function does not return a value.
+*/
 void leds_set_color(uint16_t index, uint8_t red, uint8_t green, uint8_t blue);
 
+
 ////////////////////////////// Switches/Buttons ///////////////////////////////
+
+/*
+*  This function returns the state of a switch.
+*  The switch_idx is an integer between 1 and 2, representing the number of the target switch (for example, 1 corresponds to switch 1 on the board).
+*  The bool return type means that this function returns a boolean value (true or false).
+*  True corresponds to the switch being on, and false corresponds to the switch being off.
+*/
 bool switches_get(uint8_t switch_idx);
+
+/*
+*  This function returns the state of a button.
+*  The button_idx is an integer between 1 and 3, representing the number of the target button (for example, 1 corresponds to button 1 on the board).
+*  The bool return type means that this function returns a boolean value (true or false).
+*  True corresponds to the button being pressed, and false corresponds to the button being released.
```

Comment:
Have you verified this is true? The code might be do the opposite. If so, we should fix the source code. Can you double check?

Reply:
Fixed